sceptre_cdk_handler/command_checker.py

```diff
@@ -0,0 +1,50 @@
+import shutil
+import subprocess
+from logging import Logger
+
+
+class CommandChecker:
```

Comment:
actually just noticed there is a builtin python util that already does this, `shutil`.  maybe use that instead?  https://stackoverflow.com/questions/11210104/check-if-a-program-exists-from-a-python-script

Reply:
If you look, I actually _am_ using shutil.which. however, by abstracting it into this dependency, I've made the code easier to test and abstracted away implementation details.